hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -134,6 +148,24 @@
         }
       ]
     },
+    { name: "MODULE_ENABLE",
+      desc: "Module enable register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGWEN_ME",
+      tags: [// Exclude from writes to these field because they cause side affects.
+             "excl:CsrAllTests:CsrExclAll"]
+      fields: [
+        { bits: "3:0",
```

Comment:
This fleid needs a name.

Reply:
It's turns out that a signal field name for a register does nothing. The pkg defines for hardware do not use it, and the software generated defines do not use it. However, it was omitted as an experiment to get CI working, and will be restored for consistent with other hjson files.

Reply:
The field name is now added - DONE.

Reply:
@matuten are we good here? If not, let me know what is missing.